torch/fx/interpreter.py

```diff
@@ -148,6 +149,8 @@ def run_node(self, n : Node) -> Any:
         Returns:
             Any: The result of executing ``n``
         """
+        self.current_node = n
```

Comment:
I'd prefer this to be dynamically scoped: so you save the current value of `current_node`, set it, and then on return (finally) you restore to old value

Reply:
This is mostly helpful if you have some reentrant execution of the same interpreter. Which hopefully is pretty unlikely...

Reply:
Updated per request.